src/tests/htsjdk/samtools/CRAMExceptionTest.java

```diff
@@ -0,0 +1,38 @@
+package htsjdk.samtools;
```

Comment:
Can you move `CRAMEdgeCasesTest` back to its original location in `src/tests/java/htsjdk/samtools`, move the test case in this file (`CRAMExceptionTest`) into `CRAMEdgeCasesTest`, and then delete this file?